src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowPredicate.java

```diff
@@ -491,15 +496,156 @@ private static boolean getPredicateForResponseTypeSpec(CaseData caseData, Respon
                 predicate = basePredicate;
                 break;
             case TWO_V_ONE:
-                predicate = responseType.equals(caseData.getClaimant1ClaimResponseTypeForSpec())
-                    && responseType.equals(caseData.getClaimant2ClaimResponseTypeForSpec());
+                if (YES.equals(caseData.getDefendantSingleResponseToBothClaimants())) {
+                    predicate = responseType.equals(caseData.getRespondent1ClaimResponseTypeForSpec());
+                } else {
+                    predicate = responseType.equals(caseData.getClaimant1ClaimResponseTypeForSpec())
+                        && responseType.equals(caseData.getClaimant2ClaimResponseTypeForSpec());
+                }
                 break;
             default:
                 break;
         }
         return predicate;
     }
 
+    public static final Predicate<CaseData> divergentRespondWithDQAndGoOfflineSpec = caseData ->
+        isDivergentResponsesWithDQAndGoOfflineSpec(caseData);
+
+    private static boolean isDivergentResponsesWithDQAndGoOfflineSpec(CaseData caseData) {
+
+        if (!SPEC_CLAIM.equals(caseData.getSuperClaimType())) {
+            return false;
+        }
+
+        switch (getMultiPartyScenario(caseData)) {
+            case ONE_V_TWO_ONE_LEGAL_REP:
+                //scenario: only one of them have submitted full defence response
+                return caseData.getRespondent1ClaimResponseTypeForSpec() != null
+                    && !caseData.getRespondent1ClaimResponseTypeForSpec()
+                    .equals(caseData.getRespondent2ClaimResponseTypeForSpec())
+                    && NO.equals(caseData.getRespondentResponseIsSame())
+                    && (RespondentResponseTypeSpec.FULL_DEFENCE
+                    .equals(caseData.getRespondent1ClaimResponseTypeForSpec())
+                    || RespondentResponseTypeSpec.FULL_DEFENCE
+                    .equals(caseData.getRespondent2ClaimResponseTypeForSpec()));
+            case ONE_V_TWO_TWO_LEGAL_REP:
+                //scenario: latest response is full defence
+                return caseData.getRespondent1ClaimResponseTypeForSpec() != null
+                    && caseData.getRespondent2ClaimResponseTypeForSpec() != null
+                    && caseData.getRespondent1ResponseDate() != null
+                    && caseData.getRespondent2ResponseDate() != null
+                    && !caseData.getRespondent1ClaimResponseTypeForSpec()
+                    .equals(caseData.getRespondent2ClaimResponseTypeForSpec())
+                    && ((RespondentResponseTypeSpec.FULL_DEFENCE
```

Comment:
Maybe the following is easier to understand
FULL_DEFENCE.equals(caseData.getRespondent2ResponseDate().isAfter(caseData.getRespondent1ResponseDate())?caseData.getRespondent2ClaimResponseTypeForSpec() : caseData.getRespondent1ClaimResponseTypeForSpec))

Reply:
Left to a less time-pressed occasion